recipes/ctc-segmentation/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "ctc-segmentation" %}
+{% set version = "1.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ctc_segmentation-{{ version }}.tar.gz
+  sha256: 17c3171fb7a7bbfcfacc10313398dbd761a31d3f13d3c5ea7d567e7f375f0223
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy >=1.19
```

```suggestion
    - numpy
```
We pin numpy globally.